java-desktop/src/main/java/com/couchbase/todo/controller/TaskListController.java

```diff
@@ -228,8 +228,9 @@ private void updateTaskName(@NotNull String id, @NotNull String name) {
     private void updateTaskComplete(@NotNull String id, boolean completion) {
         Document doc = DB.get().getDocument(id);
         if (doc == null) { return; }
-        System.out.println("Task doc after update complete check box: " + doc.toJSON());
-
+        if (MainController.jsonBoolean.get() == true) {
```

Comment:
Yeah... you don't need the "== true".  Didn't the IDE mention that too you?

You can just say:
if ((MainController.jsonBoolean.get()) {